.env.dist

```diff
@@ -4,3 +4,6 @@ SENTRY_ORG=nodefactory
 SENTRY_PROJECT=chainguardian
 SENTRY_AUTH_TOKEN=xxx
 DOCKER_LIGHTHOUSE_IMAGE=sigp/lighthouse:v1.1.0
+DISCORD_URL=https://discord.gg/4GBwH52cFb
+CHAINGUARDIAN_URL=https://chainguardian.nodefactory.io/
+GITHUB_URL=https://github.com/NodeFactoryIo/ChainGuardian
```

Comment:
I think you can hardcode those  in js. It doesn't change between environments

Reply:
agree only on Github URL, but for discord and app URL is good to have in ENV in case of change

Reply:
But even if it change, we would have to make new release anyways?

Reply:
yes